install.sh

```diff
@@ -26,6 +26,7 @@ function set_global_default_variable {
    reactclientport="8081"
    c2serverip=""
    c2serverport="9000"
+   redirectorport="9000"
```

Comment:
We just need a default port for docker bake. This is static, but using the docker proxy one could use whatever they want (included in the redirector install script via API)